storage/testsuite/test_queue.go

```diff
@@ -17,9 +17,9 @@ func RunQueueTests(t *testing.T, q storage.Queue) {
 
 func testBasic(t *testing.T, q storage.Queue) {
 	err := q.Enqueue(storage.Value("hello"))
-	assert.NotNil(t, err)
+	assert.Nil(t, err)
```

Comment:
Btw. there's a `assert.NoError` and `assert.Error` in the package.

Reply:
Using NoError and Error now